workshops/views.py

```diff
@@ -1349,10 +1349,7 @@ def event_import(request):
     This is used to read metadata from workshop website and then fill up fields
     on event_create form."""
 
-    # TODO: remove POST support completely
-    url = request.POST.get('url', '').strip()
-    if not url:
-        url = request.GET.get('url', '').strip()
+    url = request.GET.get('url', '').strip()
```

Comment:
I understand that this change was suggested by my "TODO" some time ago, but what actually made you finally remove the POST support?